src/public/target/sections/Safety/index.js

```diff
@@ -1,6 +1,23 @@
+import { loader } from 'graphql.macro';
+import gql from 'graphql-tag';
+
 export const id = 'safety';
 export const name = 'Safety';
 
+export const hasSummaryData = data => data;
```

Comment:
Another functional issue is that the section says ‘available’ but lists empty tables if a target has only toxicity information: unless the `safety` object is `null` entirely, the requested fields will come back as `{ "adverseEffects": [], "safetyRiskInfo": [], "__typename": "Safety" }`.

I do plan to add the safety table once this patch has been reviewed, but let me know if you think availability should be defined in a more forwards-compatible way.

Reply:
Good point @fedde-s. Since this is only alpha for the moment and you're planning on adding the toxicity table soon, I'd say it's OK as it's only temporary while still work in progress.